src/main/java/org/terasology/physicalstats/system/PhysicalStatsSystem.java

```diff
@@ -59,12 +59,13 @@ public void initialise() {
             // For every entity that has a health component, set their max health equal to CON * 10.
             if (clientEntity.hasComponent(HealthComponent.class)) {
                 HealthComponent h = clientEntity.getComponent(HealthComponent.class);
+                PhysicalStatsComponent p = clientEntity.getComponent(PhysicalStatsComponent.class);
 
-                // This check is in place so that the current health isn't reset to max health when reloading a game
-                // save. It should only be done when the two are already equal.
-                if (h.currentHealth == h.maxHealth) {
-                    h.maxHealth = clientEntity.getComponent(PhysicalStatsComponent.class).constitution * 10;
-                    h.currentHealth = h.maxHealth;
+                // Ensure that the health is set to the appropriate percentage of the appropriate maximum health
+                if (h.maxHealth != p.constitution * 10) {
```

Comment:
Please make the `10` multiplier a constant field

Reply:
Done so in commit [bf7a296](https://github.com/Terasology/PhysicalStats/pull/4/commits/bf7a29667377f79ceab442d972541a21b250a96f)